workflows/trigger-lp-build.py

```diff
@@ -100,7 +93,22 @@ def main(argv):
     primary_archive = ubuntu.getArchive(name='primary')
 
     snap = None
-    if ephemeral_build:
+    ephemeral_build = False
+    if args['git_repo_branch'] is not None and \
+       args['git_repo_branch'].startswith('snap-'):
+        # We remove the temporal branch suffix
+        rec_suffix = re.sub('_.*', '', args['git_repo_branch'])
+        build_name = "%s-%s" % (args['snap'], rec_suffix)
+        print('Getting snap recipe {} from {} team'.format(build_name, team))
```

Comment:
Perhaps we can gradually start switching the new code to use `f"..."` strings. They're available from 3.6, so hopefully supported everywhere where we could run these helpers.